### DISCUSSION TITLE
isar.objects return type is IsarCollection

### DISCUSSION THREAD

**Discussion by dgandhi17** [isar/isar#63] [Help]
Is it possible to cast it to dart type data structure ? as I don't want to return IsarCollection as any filter operation returns list of data and only this one return isarColllection. 

Can you tell me what is the benefit of this one ?

**Comment by simc** [comment]
@dgandhi17 I'm not sure I understand what you mean. There is no Isar.objects getter.

**Reply by dgandhi17** [reply]
I mean, isar.collectionname.getAll() this returns IsarCollection instead of list or other dart data structure

**Reply by simc** [reply]
The getAll method returns a list of objects as you can see here: https://github.com/isar/isar/blob/b00d2d2104eba78d72839a28501ec9ca912250f6/packages/isar/lib/src/isar_collection.dart#L10

**Reply by dgandhi17** [reply]
Sorry my bad. isar.collectionname only.
